api.md

```diff
@@ -1824,21 +1824,23 @@ ORDER BY day;
 
 ## timescaledb_information.hypertable [](timescaledb_information-hypertable)
 
-Get information about hypertables.
+Get information about hypertables. If the hypertable is distributed, the
+hypertable statistics reflect the sum of statistics across all distributed chunks.
 
 #### Available Columns
 
 |Name|Description|
 |---|---|
-| `schema_name` | Schema name of the hypertable. |
-| `table_name` | Table name of the hypertable. |
-| `table_owner` | Owner of the hypertable. |
-| `num_dimensions` | Number of dimensions. |
+| `table_schema` | Schema name of the hypertable |
+| `table_name` | Table name of the hypertable |
+| `table_owner` | Owner of the hypertable |
+| `num_dimensions` | Number of dimensions |
 | `num_chunks` | Number of chunks. |
-| `table_bytes` |Disk space used by hypertable |
-| `index_bytes` |Disk space used by indexes|
-| `toast_bytes` |Disk space of toast tables|
-| `total_bytes` |Total disk space used by the specified table, including all indexes and TOAST data|
+| `table_size` | Disk space used by hypertable |
+| `index_size` | Disk space used by indexes |
+| `toast_size` | Disk space of toast tables |
+| `total_size` | Total disk space used by the specified table, including all indexes and TOAST data|
+| `distributed` | Distributed status of the hypertable |
```

Comment:
Specify that it's a boolean?

Reply:
Flagged as boolean using the same syntax used elsewhere.